execution/execution_label.py

```diff
@@ -4,20 +4,20 @@
     text_classification_pipeline(filename='datasets/text_data.csv', target="label", predictor="feedback",
                                  test_size=0.33,
                                  ordinal=False,
-                                 tknz="spacy",
+                                 tknz="wordnet",
```

Comment:
You're switching from spacy to wordnet, is that right? Can you say why?

The licence is a bit weird, I think it's fine but worth knowing https://wordnet.princeton.edu/license-and-commercial-use

Reply:
I was just testing it out, it was already an option in the code (https://github.com/CDU-data-science-team/pxtextmining/blob/development/pxtextmining/helpers/tokenization.py) and I wanted to see how much of a difference it made (and also if it still worked with all the preprocessing changes I made). can switch it back.

Reply:
Oh, I see. Well, let's see if it makes a difference then. Just curious, I didn't know it was in there before